doc/sphinx/source/examples/Tutorials/Two_Way_Wave_Equation/Two_Way_Wave_Equation.md

```diff
@@ -0,0 +1,219 @@
+
+While each code can be run independanlty, the easiest way to compare the two methods is with `comparison_two_way_wave_md_vs_fd`
+
+---
+
+## Overview
+
+The **1D two-way wave equation** is given by:
+
+$$
+\frac{\partial^2 u}{\partial t^2} = c^2 \frac{\partial^2 u}{\partial x^2}, \quad x \in [-2, 2] \qquad \qquad \qquad (1)
+$$
+
+Both solvers use a **centered-in-time, centered-in-space (CTCS)** scheme, which is **second-order accurate in both time and space**.
+
+We assume no boundary effects (large domain), allowing a clean comparison of the spatial discretization methods. Here is a little bit more information about the two seperate functions.
+
+---
+
+## 1. `finite_diff_two_way_wave_eq`
+
+**Purpose:** This function solves the the wave equation using **standard finite differences (FD)**.
+
+**Time-stepping scheme (CTCS):**
+If we discretize the equation with standard centered second order finite differences, Equation(1) turn into 
+
+$$
+\frac{U^{k-1}_i - 2U^{k}_i + U^{k+1}_i}{\Delta t^2} = c^2\frac{U_{i+1}^k - 2U_{i}^k + U_{i+1}^k}{\Delta x^2}
+$$
+
+Rearranging the equation for the unknown value $U^{k+1}_i$ we get:
+
+$$
+U^{k+1}_i = 2U^k_i - U^{k-1}_i + \frac{c^2\Delta t^2}{\Delta x^2}\Big(U^k_{i-1} - 2U^k_{i} + U^k_{i+1} \Big)
+$$
+
+Which can be expressed as a matrix $D_{fd}$ times the vector ${U^k}$
+
+$$
+U^{k+1} = 2U^k - U^{k-1} + D_{fd}U^k
+$$
+
+
+The matrix is written out in the code using sparse matrix operations and is equivalent to:
+
+$$
+D_{fd} = \frac{c^2\Delta t^2}{\Delta x^2}
+\begin{bmatrix}
+-2 & 1 & 0 & \cdots \\
+1 & -2 & 1 & \cdots \\
+0 & 1 & -2 & \cdots \\
+\vdots & \vdots & \vdots & \ddots
+\end{bmatrix}
+$$
+
+The new values are updated each time step with values from the previous two time steps (k, k-1).
+
+$$
+U^{k+1} = 2U^k - U^{k-1} + D_{fd}U^k
+$$
+
+### Function Outputs
+| Variable | Description |
+|-----------|--------------|
+| `U2_fd` | Final solution at the last time step |
+| `error_fd` | Norm of error vs. analytic/reference solution |
+| `walltime_fd` | Wall-clock time (seconds) |
+| `flops_fd` | Estimated floating-point operation count |
+
+### Notes
+We are using an explicit two-step scheme (leapfrog). The CFL condition must be satisfied: $ c \Delta t^2 / \Delta x^2 \le 1 $. If you change the stepping, or increase the number of cells, be sure to change this value. No boundary conditions are applied, the domain is large enough to avoid reflection effects. This is to test just the spacial scheme without any boundary considerations.
+
+---
+
+## 2. `mimetic_diff_two_way_wave_eq`
+
+**Purpose:** Solve the wave equation using **mimetic difference operators (MD)**.
+
+The mimetic Laplacian operator `L` replaces the explicit finite-difference stencil, automatically enforcing discrete analogs of conservation and symmetry properties.
```

Comment:
I would remove "explicit" here, as it is often used in regards to time-stepping, but we are talking about the spatial discretization here.
```suggestion
The mimetic Laplacian operator `L` replaces the finite-difference stencil, automatically enforcing discrete analogs of conservation and symmetry properties.
```